browser/src/control/Control.ContextMenu.js

```diff
@@ -251,6 +266,22 @@ L.Control.ContextMenu = L.Control.extend({
 		}
 	},
 
+	_toSnakeCase: function (text) {
+		return text.replace(/[ _]/gi, '-').replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
+	},
+
+	_getContextMenuIcon: function(command, commandName) {
+		if (!this.options.commandsWithIcons.includes(commandName)) {
+			return '<span class="context-menu-icon-spacer"></span>';
+		}
+
+		const iconName = app.LOUtil.getIconNameOfCommand(command);
+		const iconUrl = app.LOUtil.getImageURL(iconName);
+		const imgId = 'context-menu-icon-' + this._toSnakeCase(iconName);
```

Comment:
why do we need that "toSnakeCase" ? better to have less regexp than more :)

Reply:
I removed it for the Id  in the image as we already have it in the classname through the <a>  tag 
https://github.com/CollaboraOnline/online/pull/10351